leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -275,14 +275,13 @@
   },
 
   CLIPBOARD_UI_MANAGER__SINSTANCE(
-      SAMSUNG.equals(MANUFACTURER) && SDK_INT >= KITKAT && SDK_INT <= LOLLIPOP) {
```

Comment:
1) Not sure, I will lower the check back to KITKAT
2) I'm not, but I have no device to check.